articles/fin-ops-core/fin-ops/get-started/help-overview.md

```diff
@@ -163,16 +163,12 @@ The following table lists our websites. Sites that have an asterisk (\*) next to
 
 | Site | Description |
 |------|-------------|
-| [Docs.microsoft.com](/dynamics365/) | This site hosts or links to all product documentation for Dynamics 365. |
+| [Microsoft Dynamics 365 documentation](/dynamics365/) | This site hosts or links to all product documentation for Dynamics 365. |
 | [Microsoft Learn](/learn/) | This site is the free Microsoft eLearning site. |
 | [Microsoft Dynamics Lifecycle Services (LCS)](https://lcs.dynamics.com/)\* | This site provides a cloud-based collaborative workspace that customers and partners can use to manage projects from pre-sales to implementation and operations. It's useful in all phases of an implementation. |
 | [Support blog](https://aka.ms/AXSupportBlog) | This site provides tips and tricks that are posted by the Support team. |
-| [Docs.microsoft.com/previous versions](/previous-versions/dynamics/) | This site hosts content from previous releases. |
+| [[](/previous) versions](/previous-versions/dynamics/) | This site hosts content from previous releases. |
```

Comment:
Previous versions should be in one set of brackets. [Previous versions](/previous-versions/dynamics/)